src/AutoRest.CSharp.V3/Output/Builders/BuilderHelpers.cs

```diff
@@ -82,6 +82,17 @@ public static TypeDeclarationOptions CreateTypeAttributes(string defaultName, st
             );
         }
 
+        public static MemberDeclarationOptions CreatMemberDeclaration(string defaultName, CSharpType defaultType, Accessibility defaultAccessibility, ISymbol? existingMember)
+        {
+            // Not reading the return type information of existing members yet
```

Comment:
I could live without that for a long while.  Just being able to make one or two fields internal that I expose differently would be a big improvement.